dev/src/__tests__/updateEnrollment.test.tsx

```diff
@@ -0,0 +1,92 @@
+import { Enrollment, User } from "../../proto/ag/ag_pb"
+import { createOvermindMock } from "overmind"
+import { config } from "../overmind"
+import { configure, render } from "enzyme"
+import Adapter from "@wojtekmaj/enzyme-adapter-react-17"
+import { createMemoryHistory } from "history"
+import React from "react"
+import Members from "../components/Members"
+import { Route, Router } from "react-router"
+import { Provider } from "overmind-react"
+import { MockGrpcManager } from "../MockGRPCManager"
+
+
+React.useLayoutEffect = React.useEffect
+
+describe("UpdateEnrollment", () => {
+    const mockedOvermind = createOvermindMock(config, {
+        grpcMan: new MockGrpcManager()
+    })
+    it("Pending student gets accecpted", async () => {
+        await mockedOvermind.actions.getEnrollmentsByCourse({ courseID: 2, statuses: [] })
+        //This is a user with status pending
```

Comment:
Please use space between comment symbol and text: `// This ...`

Reply:
I'm not into the frontend code... but I'm struggling to see the `user with status pending` in the code. The same applies to the other comments below. Can you clarify the comment please.

Reply:
status refers to the userStatus in class Enrollment. 
`  export enum UserStatus { 
    NONE = 0,
    PENDING = 1,
    STUDENT = 2,
    TEACHER = 3,
  }`